addon/styles/power-select-variables.scss

```diff
@@ -0,0 +1,7 @@
+.ember-power-select-trigger {
+  border-top: 0;
+  border-left: 0;
+  border-bottom: 0;
+  border-right: 0;
+}
+@import 'ember-power-select';
```

Comment:
I could be mistaken, but it doesn't look like this file is used. If it isn't, recommend removing.